src/theories/Monoidal.jl

```diff
@@ -65,16 +67,74 @@ ndims(expr::ObExpr) = 1
 ndims(expr::ObExpr{:otimes}) = sum(map(ndims, args(expr)))
 ndims(expr::ObExpr{:munit}) = 0
 
-function show_unicode(io::IO, expr::Union{ObExpr{:otimes},HomExpr{:otimes}}; kw...)
+function show_unicode(io::IO, expr::Union{ObExpr{:otimes},HomExpr{:otimes},HomVExpr{:otimes},HomHExpr{:otimes},Hom2Expr{:otimes}}; kw...)
```

Comment:
At this point, maybe these should just be `expr::GATExpr{:otimes}` rather than a long `Union`.

Reply:
Ah, this is a good idea. Resolved in later commit.